profiles/rgw-radosgw-static-website

```diff
@@ -0,0 +1,4 @@
+ceph_conf_overrides:
```

Comment:
you need this header: https://github.com/ceph/ceph-ansible/blob/581d9a59f40d3a7b35e5568a8ebba22066079fc9/profiles/rgw-keystone-v2#L1-L6